src/community/pom.xml

```diff
@@ -98,6 +99,11 @@
             <descriptor>release/ext-smart-data-loader.xml</descriptor>
             <descriptor>release/ext-cov-json.xml</descriptor>
             <descriptor>release/ext-schemaless-features.xml</descriptor>
+            <descriptor>release/ext-gwc-mbtiles.xml</descriptor>
+            <!-- As per our discussion on list, the module does not work, leaving it for eventual salvage,
```

Comment:
I'm pretty confused, why is this commented out section here? It was not in the original source (see diff)

Reply:
Right, it looks like it has been removed by a PR merged a couple of days ago.